ASPNET-CORE-MVC/Infrastructure/Services/UserService/UserService.cs

```diff
@@ -11,13 +15,19 @@ public class UserService : IUserService
     private readonly ILogger _logger;
     private readonly UserManager<User> _userManager;
     private readonly SignInManager<User> _signInManager;
+    private readonly IEmailSenderService _emailSender;
+    private readonly IFlashMessage _flashMessage;
+    private readonly IHttpContextAccessor _httpContextAccessor;
 
-    public UserService(ILogger logger,
-        UserManager<User> userManager, SignInManager<User> signInManager)
+    public UserService(ILogger logger, UserManager<User> userManager, 
+        SignInManager<User> signInManager, IEmailSenderService emailSender, IFlashMessage flashMessage, IHttpContextAccessor httpContextAccessor)
```

Comment:
I think it would be better to end line after every property, to just once in the beginning

Reply:
Imo end after every parameter looks a bit weird, but split on two lines by half is good idea. Done